### PR TITLE
ZK-3843: wire an event listener doesn't work for a Listitem in HtmlMa…

### PR DESCRIPTION
**Pull request by klyvechen**
…croComponent

### CODE REVIEW

File: zktest/src/archive/test2/config.properties [hunk 1]
```diff
@@ -2868,6 +2868,7 @@ F85-ZK-3875.zul=A,E,Textbox,submitByEnter
 ##ztl##F85-ZK-3538.zul=A,E,Tree,focus
 ##ztl##F85-ZK-3619.zul=A,E,Calendar,constraint
 ##zats##F85-ZK-3181.zul=A,E,onClientInfo,redundant,detach,onListenerChange
+##zats##F85-ZK-3843.zul=A,E,HTMLMacroComponent,Selector,wireComponent
```
[the remark this answers is not in the document]
Comment (klyvechen) [reply zktest/src/archive/test2/config.properties]: ok

File: zkdoc/release-note [hunk 2]
```diff
@@ -8,6 +8,7 @@ ZK 8.5.2
   ZK-3538: Do not change focus when opening a Treeitem
   ZK-3619: Calendar supports those constraints as Datebox
   ZK-3181: redundant requests filtering with clientInfo listeners
+  ZK-3843: wire an event listener doesn't work for a Listitem in HtmlMacroComponent
```
[the remark this answers is not in the document]
Comment (klyvechen) [reply zkdoc/release-note]: ok

File: zk/src/org/zkoss/zk/ui/HtmlMacroComponent.java [hunk 3]
```diff
@@ -169,7 +171,15 @@ protected void compose() {
 
 			exec.createComponents(_uri != null ? _uri : getDefinition().getMacroURI(), this, _props);
 		}
+		this.addEventListener(-10000, "onCreate", new EventListener<Event>() {
```
[the remark this answers is not in the document]
Comment (klyvechen) [reply zk/src/org/zkoss/zk/ui/HtmlMacroComponent.java]: ok

File: zktest/src/archive/test2/config.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (scribetw) [reply zktest/src/archive/test2/config.properties]: Not ok. No changes

[the remark this answers is not in the document]
Comment (klyvechen) [reply zktest/src/archive/test2/config.properties]: ok

**Comment by wenninghsu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**